src/SignalR/common/Http.Connections/src/Internal/HttpConnectionManager.cs

```diff
@@ -31,12 +32,14 @@ public partial class HttpConnectionManager
         private readonly ILogger<HttpConnectionManager> _logger;
         private readonly ILogger<HttpConnectionContext> _connectionLogger;
 
-        public HttpConnectionManager(ILoggerFactory loggerFactory, IApplicationLifetime appLifetime)
+        public TimeSpan DisconnectTimeout { get; set; }
+
+        public HttpConnectionManager(ILoggerFactory loggerFactory, IApplicationLifetime appLifetime, IOptions<ConnectionOptions> connectionOptions)
```

Comment:
This is a breaking change but I'm not sure how much we care. Add another constructor until we decide it's OK to break here?